drivers/mfd/mfd_ite_it8801.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_it8801_mfd
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/mfd/mfd_ite_it8801.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(mfd_ite_it8801, CONFIG_MFD_LOG_LEVEL);
+
+static const struct device *kbd_dev = DEVICE_DT_GET_ANY(ite_it8801_kbd);
```

Comment:
This driver supports multiple instances, but this this assumes that there is only one IT8801 defined.

Reply:
Fixed, use multiple instances.